### ISSUE TITLE
banners for each folders

### ISSUE THREAD

**Issue by mckaygerhard** [dharmann/info_banner#1]
i tested u'r simgle plugin and its very ussefully·..

i want to ask how can i made this feature .. 

for each specific folder display diferents banners..

in the js file i have the selection for folders, but then invoked and callback (i dont know so much about javascript and rc framework)

i can display a diferent banner for each folder.. 

by example, the message for spam are very good, but for inbos remenber about full storage...

**Comment by dharmann** [comment]
Specific banner for every folder require a little bit more of work. You need to build a js function to change showed message for specific folder. And if you need more languages support ,maybe, you need to write a js dictionary to translate client side the desired message. In any case, I think you need to work on client side functions, because the php side it's needed only as trigger.
I'm sorry if I'm not accurate enough in my explanation, but I wrote this code some time ago...
In any case, I'm at disposition for more insights... For now, merry christmas and happy new year!!!! ;)

**Comment by mckaygerhard** [comment]
yeah i note that.. i still learning the roundcube code i'll try to work in that and commit something here.. 

well merry "commercial" crishmas too...

**Comment by mckaygerhard** [comment]
wow two years since that! well i made to work only for specific direcories .. merry "commercial" christmas again and happy new 2020 year, i'll try to make it work in recent versions. if not see at roundcubevnz group
